apis/nucleus/src/components/listbox/__tests__/list-box.test.jsx

```diff
@@ -253,7 +256,7 @@ describe('<Listbox />', () => {
     test('should prevent InfiniteLoader to get itemCount == 0', async () => {
       layout.qListObject.qSize.qcy = 0;
       await render();
-      expect(infiniteProps.itemCount).toBe(1);
+      expect(infiniteProps.itemCount).toBe(2);
```

Comment:
This happens because it's testing the previous test case: It never renders the `<InfiniteLoader />` since we are showing a disclaimer instead. Since we are showing a text "No matches for your terms." we need to verify that this only happens when we:
1. Search for something
2. We get no matches

Are there no other cases where we might end up with `listCount === 0`?

Reply:
I cannot see any problems with this right now. I will merge this.